ui/imports/shared/views/ProfileDialogView.qml

```diff
@@ -182,6 +182,22 @@ Pane {
         }
     }
 
+    Component {
+        id: btnRevertContactRequestRejectionComponent
+
+        StatusButton {
+            size: StatusButton.Size.Small
+            text: qsTr("Reverse Contact Rejection")
+            icon.name: "refresh"
+            icon.width: 16
```

Comment:
@caybro @alexandraB99 maybe its better to make default icon width and height depend size in StatusBaseButton?

Reply:
Good idea (imho), let's discuss in a separate issue?